wordpress-mobile/AztecEditor-iOS
# Enforce HTML Paragraphs where no block-level element exists.

### Description:

Fixes #667 

Enforces all content to exist inside block-level elements, defaulting to a paragraph when there's no parent block-level element.

This PR also simplifies the EoL conversion algorythm:

 - Newlines / line feeds are converted to an interruption of block-level elements.
- Line separators are converted to BRs.

### How to test:

- Run the Sample app.
- Convert back and forth and make sure the newlines are properly converted.
- Also make sure paragraphs are enforced on content without any block-level parent element.
- Press ENTER or SHIFT + ENTER while editing in visual mode, and see the difference when switching to HTML mode.
